File: dmn-test-cases/standard/proto/1.1/date-time-proto/translator/expected/proto3/java/dmn/ModelElementRegistry.java

public class ModelElementRegistry extends com.gs.dmn.runtime.discovery.ModelElementRegistry {
    public ModelElementRegistry() {
        // Register elements from model 'proto-date-time-function'
        register("CompositeDateTime", "CompositeDateTime");
        register("Date", "Date");
        register("DateTime", "DateTime");
        register("Time", "Time");
    }
}

File: dmn-test-cases/standard/tck/1.1/cl2/0007-simpletable-P2/translator/expected/kotlin/dmn/ModelElementRegistry.kt

class ModelElementRegistry : com.gs.dmn.runtime.discovery.ModelElementRegistry {
    constructor() {
        // Register elements from model 'simple P table 2'
        register("Age", "Age")
        register("'Approval Status'", "ApprovalStatus")
        register("RiskCategory", "RiskCategory")
        register("isAffordable", "IsAffordable")
    }
}
